flow/packet-headers/arp.yaml

```diff
@@ -0,0 +1,58 @@
+openapi: 3.1.0
+
+info:
+  title: Arp packet header model
+  version: ^0.0.0
+
+components:
+  schemas:
+    Flow.Arp:
+      description: Arp packet header
```

Comment:
Change to ARP instead of Arp in the description.

Reply:
This (all caps in description) will require change for other protocol descriptions as well. e.g. https://github.com/open-traffic-generator/models/blob/master/flow/packet-headers/udp.yaml#L10

(I agree with the change)

Reply:
I didn't look closely at the other headers.  Glanced at a couple and they seemed ok.  Regardless, it should be ARP in the description.

Reply:
Updated description.